content/applications/finance/accounting/fiscal_localizations/localizations/italy.rst

```diff
@@ -2,78 +2,382 @@
+  under :menuselection:`General Settings --> Companies (section)`;
+- :guilabel:`VAT`: the **buyer**'s VAT/TIN number (on their profile card);
+- :guilabel:`Total`: the total **amount** (VAT included) of the invoice.
+
+In the :abbr:`EDI (Electronic Data Interchange)`, Odoo exports invoices as simplified if:
+
+- It is a **domestic** transaction (i.e., the partner is from Italy);
+- The buyer's data is **insufficient** for a regular invoice;
+- The **required fields** for a regular invoice (address, ZIP code, city, country) are provided;
+- The total amount VAT included is **less** than **400 EUR**.
+
+Invoices can have different :guilabel:`Tipo Documento`, depending on the move type:
+
+- `TD07`: Simplified invoice;
+- `TD08`: Simplified credit note.
+
+Deferred invoice
+----------------
+
+The **deferred invoice** is an invoice that is **issued at a later time** than the sale of goods or
+the provision of services. A **deferred invoice** has to be issued at the latest within the **15th
+day** of the month following the delivery covered by the document.
+
+It usually is a **summary invoice** containing a list of multiple sales of goods or services,
+carried out in the month. The business is allowed to **group** the sales into **one invoice**,
+generally issued at the **end of the month** for accounting purposes. Deferred invoices are default
+for **wholesaler** having recurrent clients.
+
+If the goods are transported by a **carrier**, every delivery has an associated **Documento di
+Transporto (DDT)**, or **Transport Document**. The deferred invoice **must** indicate the details of
+all the **DDTs** information for better tracing.
+
+.. note::
+   e-Invoicing of deferred invoices requires the `l10n_it_stock_ddt` :ref:`module <italy/modules>`.
+   In this case, a dedicated :guilabel:`Tipo Documento` `TD24` is used in the e-Invoice.
+
+Odoo exports moves as `TD24` if the following conditions are met:
+
+- Is an invoice;
+- Is associated to deliveries whose **DDTs** have a **different** date than the issuance date of the
+  invoice.
+
+Vendor bills
+============
+
+When creating a **vendor bill**, the :guilabel:`FatturaPA` feature is enabled by default and
+automatically activated under :menuselection:`Accounting --> Configuration --> Journals --> Vendor
+Bills (journal) --> Advanced Settings (tab) --> Electronic Data Interchange`.
+
+When inserting taxes in a vendor bill, you have the possibility to choose **Reverse Charge** taxes,
+these are automatically activated in the Italian fiscal position. By going to
+:menuselection:`Configuration --> Taxes`, you will see that the 10% and 22% services and goods taxes
+are activated and have the correct tax grids; these are set up automatically to ensure the correct
+booking of accounting entries and display of the tax report.
+Once the bill is confirmed, a banner appears, and the bill can be sent to the EDI service.
```

Comment:
"A banner appears" is very generic.
It's the same banner as the other EDI, I don't know if you have information about them to be lined. Otherwise, just skip it.